### ISSUE TITLE
Attention and Latent Radio Buttons both go blank in Img2Img

### ISSUE THREAD

**Issue by blugail** [hako-mikan/sd-webui-regional-prompter#284]
Both the "Attention" and "Latent" radio buttons clear themselves when using regional prompter in Img2Img (Automatic 1111)
I think it always defaults back to "Attention" mode. (but I can't tell for sure as both buttons are empty.)

**Environment**
Web-UI version: Automatic 1111 version: [v1.7.0]
I tried it with many models including dreamshaper8 and realcartoon3d, as well as some XL models (artium, realcartoon-xl)

**Other Enabled Extensions**
None.

**Comment by blugail** [comment]
Here's a good example of the problem of using attention mode in img2img. It tends to give everyone the same face.

Txt2Img output:
![00252-3206455380](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/136867112/257d66ac-54da-449c-a853-8e7ea9c1dc22)

After Img2Img (notice the boy and the woman now have the same nose, eyes and lips)
![00038-2163213852](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/136867112/dc0f3806-2ae3-4f7a-8578-4c064072a9c8)

**Comment by pixlemedia** [comment]
Same issue here (not only in img2img but also in txt2img using SD.Next UI under MacOS M2).
Downgrade possibility would be appreciated.

**Comment by AIDungeonTester2** [comment]
I also noticed this issue. The metadata of the image says "Latent", but I believe that despite that Attention is still being used.

When I tried using img2img in Latent mode, one character LoRA for each column, every column was being affected by the third column's LoRA (and likely by the others as well, but to a lesser extent).

To test this, I tried enabling CFG Rescale to exploit the bug described here https://github.com/hako-mikan/sd-webui-regional-prompter/issues/278. If Latent was being used, it should give me grey areas for each column except the last one, similar to the issue caused by CFG Rescale.

It did not cause this issue, and still applied every LoRA to each column, further strengthening my belief that Attention is being used, no matter what you choose for img2img.
